omniduct/remotes/ssh.py

```diff
@@ -16,8 +16,17 @@
 from omniduct.remotes.base import RemoteClient
 from omniduct.utils.debug import logger
 from omniduct.utils.processes import run_in_subprocess
+from omniduct.utils.config import config
 from omniduct.errors import DuctAuthenticationError
 
+
+config.register(
```

Comment:
This should probably be handled by a configuration passed to the SSHClient constructor, rather than a magic configuration key; since some clients may need to be stricter than others.